packages/core/ios/RNSentry.mm

```diff
@@ -161,6 +161,22 @@ - (SentryOptions *_Nullable)createOptionsWithDictionary:(NSDictionary *_Nonnull)
             sentryOptions.integrations = integrations;
         }
     }
+    
+    // Set spotlight option
+    if ([mutableOptions valueForKey:@"spotlight"] != nil) {
+        id spotlightValue = [mutableOptions valueForKey:@"spotlight"];
+        if ([spotlightValue isKindOfClass:[NSString class]]) {
+            NSLog(@"The value is a string: %@", spotlightValue);
```

Comment:
Suggestion: Let's specify that we are using this string for Spotlight, so users don't get confused when reading the logs.

```suggestion
            NSLog(@"Using Spotlight on address: %@", spotlightValue);
```

Reply:
Good catch Lucas 🙇

Reply:
I tested the branch locally and its doing what it should do :D 

This is the only issue I spotted after the review.